BasicSample/Assets/ARAnchor/Scripts/AnchorsSample.cs

```diff
@@ -216,11 +216,42 @@ public void ToggleAnchorPersistence(ARAnchor anchor)
             }
             else
             {
-                m_anchorStore.UnpersistAnchor(anchor.GetComponent<SampleAnchor>().Name);
-                Debug.Log($"Anchor forgotten: {anchor.trackableId}");
-                sampleAnchor.Name = "";
-                sampleAnchor.Persisted = false;
+                UnpersistAnchor(anchor);
             }
         }
+
+        public void ClearAnchorStore()
+        {
+            // Unpersist every anchor in the scene
```

Comment:
from Pooja:  we should call "anchor store clear" and show how app should react to such clear operation.